# try



NAME: Mohamad Saad Ansar Bijapuri
PRN: 22070123070


Title:Number system

Aim: To learn Object oriented programming in C++

Theory: Exception handling in C++ provides you with a way of handling unexpected circumstances like runtime errors. 
So whenever an unexpected circumstance occurs, the program control is transferred to special functions known as handlers.

To catch the exceptions, you place some section of code under exception inspection. The section of code is placed within the try-catch block.


Algorithm:

Start =>  declare and take input of numerator and denominator => then using try if throw catch statements we check whether the division can take place => if it can take place then division is done and output is printed, else error is shown =>  End


Explanation of the algorithm:
In this code we declare and take input of numerator and denominator. Then using try if throw catch statements we check whether the division can take place. If it can take place then division is done and output is printed, else error is shown.
